secp256k1-sys/src/lib.rs

```diff
@@ -1098,6 +1239,51 @@ impl <T: CPtr> CPtr for Option<T> {
     }
 }
 
+pub const MUSIG_KEYAGG_LEN: usize = 197;
+pub const MUSIG_SECNONCE_LEN: usize = 132;
+pub const MUSIG_PUBNONCE_LEN: usize = 132;
+pub const MUSIG_AGGNONCE_LEN: usize = 132;
+pub const MUSIG_AGGNONCE_SERIALIZED_LEN: usize = 66;
+pub const MUSIG_PUBNONCE_SERIALIZED_LEN: usize = 66;
+pub const MUSIG_SESSION_LEN: usize = 133;
+pub const MUSIG_PART_SIG_LEN: usize = 36;
```

Comment:
```suggestion
pub const MUSIG_PART_SIG_LEN: usize = 36;
pub const MUSIG_PART_SIG_SERIALIZED_LEN: usize = 32;
```

FTR, I previously suggested changing MUSIG_PART_SIG_LEN to 32 but that is wrong. It really holds 36 bytes in memory, however, it is 32 bytes in serialized form. Therefore need for another constant.

Reply:
Comments on the constants explaining this would be great too.

Reply:
Good catch. Fixed the serialized size and also added comments explaining the constants.